plugins/star-rating/frontend/public/stylesheets/countly-feedback.css

```diff
@@ -0,0 +1,526 @@
+@import url('https://fonts.googleapis.com/css?family=Lato');
```

Comment:
We have some offline clients, or clients from China, without access to google, thus it would be great to have fonts packed together, like here:
https://github.com/Countly/countly-server/tree/master/frontend/express/public/fonts

is it possible to load fonts from countly server instead?